src/App/Mode.php

```diff
@@ -25,6 +25,7 @@
 use Friendica\Core\Config\ValueObject\Cache;
 use Friendica\Database\Database;
 use Friendica\Util\BasePath;
+use phpDocumentor\Reflection\Types\Static_;
```

Comment:
I'm sorry what?

Reply:
hehe, sometimes I start typing `stat..` and hit the "Tab" Button, and voliá, the IDE thinks importing `Static_` seems the right guess ;-)

Reply:
normally I don't forget to remove it again .. not this time :D